gn3/api/metadata_api/wiki.py

```diff
@@ -64,8 +64,8 @@ def edit_wiki(comment_id: int):
                 category_addition_query, (comment_id,
                                           insert_dict["versionId"], cat_id)
             )
-        return jsonify({"success": "ok"})
-    return jsonify(error="Error editting wiki entry, most likely due to DB error!"), 500
+        return jsonify({"success": "ok"}), 201
```

Comment:
nitpick: I don't mind this but the api we call is `edit_wiki` and I think the expectation is that this modifies  the comment. The fact that we add an extra row is an implementation detail since we could end up changing this to run an `update` instead. Can we do `200` instead?